### DISCUSSION TITLE
Ban players from claims

### DISCUSSION THREAD

**Discussion by Hedwig7s** [GriefPrevention/GriefPrevention#1858] [Ideas]
Ability for owners of a claim to ban people from entering the claim. Be useful if you have someone who keeps going into your claim without permission.

**Comment by Jikoo** [comment]
[GPFlags has a flag for that](https://github.com/akdukaan/GriefPreventionFlags/wiki/Flags#noenterplayer). As far as base GP is concerned, if you don't want players to enter your base, you should secure it by sealing entrances with some form of door that requires trust to use.

**Reply by Hedwig7s** [reply]
> [GPFlags has a flag for that](https://github.com/akdukaan/GriefPreventionFlags/wiki/Flags#noenterplayer). As far as base GP is concerned, if you don't want players to enter your base, you should secure it by sealing entrances with some form of door that requires trust to use.

It isn't just the base
It is land that I own in general
I'll ask the server admin if they can look into it
